skimage/draw/draw3d.py

```diff
@@ -62,18 +63,24 @@ def ellipsoid(a, b, c, spacing=(1.0, 1.0, 1.0), levelset=False):
 
 
 def ellipsoid_stats(a, b, c):
-    """
-    Calculates analytical surface area and volume for ellipsoid with
-    semimajor axes aligned with grid dimensions of specified `spacing`.
+    """Calculate analytical volume and surface area of an ellipsoid.
+
+    The surface area of an ellipsoid is given by
+
+    .. math:: S=4\\pi b c R_G\\!\\left(1, \\frac{a^2}{b^2}, \\frac{a^2}{c^2}\\right)
```

Comment:
```suggestion
    .. math:: S=4\\pi b c R_G\\!\\left(\\frac{a^2}{b^2}, \\frac{a^2}{c^2}, 1\\right)
```

Shouldn't we follow the convention of the cited source which places the $1$ at the end of $R_G(...)$?

Reply:
I think it's fine because of the complete symmetry.

Reply:
Ah, you are right, the order of the arguments doesn't matter. I'll merge then.